java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/RelationState.java

```diff
@@ -23,9 +23,9 @@
 
 /**
  * This enum values are the possible returns of the method implies(SymbolicValueRelation) of class SymbolicValueRelation.
- * @see BinaryRelation#implies
+ * @see RelationalSymbolicValue#implies(RelationalSymbolicValue)
```

Comment:
There is still a reference in the javadoc of [RelationStateTable.javaL#76](https://github.com/SonarSource/sonar-java/blob/master/java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/RelationStateTable.java#L76). Please update it.